Apps/WebClient/src/ClientApp/src/store/modules/waitlist/types.ts

```diff
@@ -27,6 +27,7 @@ export interface WaitlistGetters extends GetterTree<WaitlistState, RootState> {
 type StoreContext = ActionContext<WaitlistState, RootState>;
 export interface WaitlistActions extends ActionTree<WaitlistState, RootState> {
     getTicket(context: StoreContext): Promise<Ticket>;
+    scheduleCheckIn(context: StoreContext): Promise<void>;
```

Comment:
The 2nd parameter should be included in the signature.

Reply:
Oh yes, I forgot I changed this after defining the interface with a different approach in mind, good catch